src/routes/products/data.js

```diff
@@ -0,0 +1,114 @@
+import sardinoBlanco from '$lib/assets/bottles/sardino-blanco-big.avif'
+import sardinoRojo from '$lib/assets/bottles/sardino-rojo-big.avif'
+import forzudoRojo from '$lib/assets/bottles/forzudo-rojo-big.webp'
+import forzudoBlanco from '$lib/assets/bottles/Frozudo-Blanco_med_skygge_900x.webp'
+import carmeletaOrange from '$lib/assets/bottles/orange-carmelta-big.avif'
+import carmeletaBlanco from '$lib/assets/bottles/carmeleta-blanco-big.avif'
+import tabira from '$lib/assets/bottles/tabira-big.avif'
+import sardinoBlancoSmall from '$lib/assets/bottles/sardino-blanco.avif'
+import sardinoRojoSmall from '$lib/assets/bottles/sardino-rojo.avif'
+import forzudoRojoSmall from '$lib/assets/bottles/forzudo-rojo.avif'
+import forzudoBlancoSmall from '$lib/assets/bottles/forzudo-blanco.avif'
+import carmeletaOrangeSmall from '$lib/assets/bottles/orange-carmeleta.avif'
+import carmeletaBlancoSmall from '$lib/assets/bottles/carmeleta-blanco.avif'
+import tabiraSmall from '$lib/assets/bottles/tabira.avif'
+
+export const vermouths = [
+  {
+    brand: 'Forzudo Rojo',
+    image: forzudoRojo,
+    cardImage: forzudoRojoSmall,
+    name: 'forzudo-rojo',
```

Comment:
don't add unnecessary properties, this can be generated on the fly from the real name